src/main.rs

```diff
@@ -205,45 +231,26 @@ async fn collect_from_pods(
             };
             if let Some(requirements) = container.resources {
                 if let Some(r) = requirements.requests {
-                    for request in r
-                        .into_iter()
-                        .filter(|a| accept_resource(&a.0, resource_names))
-                    {
-                        let quantity = Qty::from_str(&(request.1).0).with_context(|| {
-                            format!(
-                                "Failed to read Qty of location {:?} / request {:?}",
-                                &location, &request
-                            )
-                        })?;
-                        resources.push(Resource {
-                            kind: request.0,
-                            usage: ResourceUsage::Requested,
-                            quantity,
-                            location: location.clone(),
-                        });
-                    }
+                    // TODO do max resource for this pod
```

Comment:
@davidB How do you want to account for the initContainers.  In logic elsewhere you assume that the pod resources all add.  initContainers break that assumption.  One simple solution is to only add one entry to `resource_names` for each pod.  This would make the container_name field obsolete.  Does that work for you or do you really want to track this at the container level and not the pod level?

Reply:
I'm in favor to remove the container level. it should ease to mimic the sheduler computation and I guess nobody display/use the container level info.

Reply:
I implemented this.